aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -1161,4 +1071,36 @@ inline void transpose_mxn(const T* src, int64_t ld_src, T* dst, int64_t ld_dst)
   }
 }
 
+#if (defined(CPU_CAPABILITY_AVX512) || defined(CPU_CAPABILITY_AVX2)) && !defined(_MSC_VER)
```

Comment:
This is necessary?

Reply:
No, it's not necessary. I prefer to add it for the function declaration because the corresponding implementation defined with same macro. I thin we can remove it also.

Reply:
I have removed it.